src/local/butler/lint.py

```diff
@@ -185,6 +185,14 @@ def execute(_):
   for file_path in py_changed_file_paths:
     _execute_command_and_track_error('pylint ' + file_path)
     _execute_command_and_track_error('yapf -d ' + file_path)
+    futurize_output = _execute_command_and_track_error(
+        'futurize -0 -x libfuturize.fixes.fix_unicode_keep_u '
```

Comment:
please put the fixes in a list var global / butler constants.py and then just add them here with -x

Reply:
Done